modules/monitoring/manifests/checks/sidekiq.pp

```diff
@@ -2,31 +2,37 @@
 #
 # Nagios alerts for sidekiq queue latency
 #
+# [*enable_signon_check*]
```

Comment:
```suggestion
# [* check_signon_queue_sizes*]
```

Reply:
But I wonder if we should be consistent with the naming here and either make them both `enable_$app_check` or `check_$app_queue_sizes`?

Reply:
Right, so I've definitely written the wrong thing somewhere, as this comment doesn't even match the thing it's meant to document.

I'm fine changing both if you think that's a better name, but initially I'll just make this new parameter consistent with the existing one.